components/src/core/components/Input/CheckboxInput.vue

```diff
@@ -47,13 +53,12 @@ export default defineComponent({
       type: String,
       default: '',
     },
-    trueValue: {
```

Comment:
Are we totally based on boolean? why `trueValue`, `falseValue` removed?

Reply:
not really, since we are binding attributes to checkbox, `trueValue` and `falseValue` props will be added if defined in outer component. I've updated [this story](https://github.com/orangehrm/oxd/pull/27/files#diff-3ca23e9ff5fb665fb82e19ec1f875146e0b7b0dfd724259e61338d20fc9cb53bR36) to better show how we can use this.